Directory.Build.props

```diff
@@ -57,7 +58,7 @@
     </None>
   </ItemGroup>
 
-  <ItemGroup Label="Build Tools" Condition="$([MSBuild]::IsOsPlatform('Windows')) and '$(TargetFramework)' != 'net8.0'">
+  <ItemGroup Label="Build Tools" Condition="$([MSBuild]::IsOsPlatform('Windows')) and (('$(TargetFramework)' != 'net8.0') or ('$(TargetFramework)' != 'net9.0'))">
```

Comment:
Shouldn't it be an and?

`"'$(TargetFramework)' != 'net8.0') or ('$(TargetFramework)' != 'net9.0'))"` is equivalent to:

`"! ('$(TargetFramework)' == 'net8.0') and ('$(TargetFramework)' == 'net9.0'))"` 

which is always true, so not sure that's you wanted to express.

Reminder:
`not A or not B` equivalent to `not (A and B)`

Reply:
good catch @jmprieur

Reply:
It should be. Thanks for catching it @jmprieur!